maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -1031,20 +1032,21 @@ private int execute( CliRequest cliRequest )
                 }
             }
 
-            if ( !failedProjects.isEmpty() )
+            List<MavenProject> sortedProjects = result.getTopologicallySortedProjects();
+            if ( result.canResume() )
+            {
+                logBuildResumeHint( "mvn <args> -r " );
```

Comment:
Why is there a trailing space?

Reply:
Nice find! I don't think it's intentional and it has not been introduced in this PR, but it does not look like any the feature or integration tests will be impacted by removing the trailing space. I'll get rid of it.